[2025-06-09T08:06:25.199+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-09T08:06:25.229+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NetSuite_Daily_ToWarehouse_F99.Download_And_Upsert_F99 manual__2025-06-09T08:05:54.563998+00:00 [queued]>
[2025-06-09T08:06:25.236+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NetSuite_Daily_ToWarehouse_F99.Download_And_Upsert_F99 manual__2025-06-09T08:05:54.563998+00:00 [queued]>
[2025-06-09T08:06:25.237+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-09T08:06:25.254+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): Download_And_Upsert_F99> on 2025-06-09 08:05:54.563998+00:00
[2025-06-09T08:06:25.261+0000] {standard_task_runner.py:63} INFO - Started process 36 to run task
[2025-06-09T08:06:25.265+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'NetSuite_Daily_ToWarehouse_F99', 'Download_And_Upsert_F99', 'manual__2025-06-09T08:05:54.563998+00:00', '--job-id', '375', '--raw', '--subdir', 'DAGS_FOLDER/dag_netsuite.py', '--cfg-path', '/tmp/tmpz9iyyk1w']
[2025-06-09T08:06:25.270+0000] {standard_task_runner.py:91} INFO - Job 375: Subtask Download_And_Upsert_F99
[2025-06-09T08:06:25.346+0000] {task_command.py:426} INFO - Running <TaskInstance: NetSuite_Daily_ToWarehouse_F99.Download_And_Upsert_F99 manual__2025-06-09T08:05:54.563998+00:00 [running]> on host cfb67c7eb5e0
[2025-06-09T08:06:25.459+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='NetSuite_Daily_ToWarehouse_F99' AIRFLOW_CTX_TASK_ID='Download_And_Upsert_F99' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T08:05:54.563998+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T08:05:54.563998+00:00'
[2025-06-09T08:06:25.462+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-09T08:06:55.414+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-09T08:06:55.415+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/dag_netsuite.py", line 36, in download_and_upsert_F99
    raise Exception("Không thể tải hoặc dữ liệu không hợp lệ")
Exception: Không thể tải hoặc dữ liệu không hợp lệ
[2025-06-09T08:06:55.428+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=NetSuite_Daily_ToWarehouse_F99, task_id=Download_And_Upsert_F99, run_id=manual__2025-06-09T08:05:54.563998+00:00, execution_date=20250609T080554, start_date=20250609T080625, end_date=20250609T080655
[2025-06-09T08:06:55.443+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 375 for task Download_And_Upsert_F99 (Không thể tải hoặc dữ liệu không hợp lệ; 36)
[2025-06-09T08:06:55.480+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-09T08:06:55.537+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-09T08:06:55.540+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
